applications/asset_tracker/src/main.c

```diff
@@ -1203,9 +1219,75 @@ static void sensors_init(void)
 }
 
 #if defined(CONFIG_USE_UI_MODULE)
+
+#if defined(CONFIG_CLOUD_TEST_LWT)
+/**@brief Prevent device reboot so cloud state can be checked manually. */
+static void halt_device(void)
+{
+	LOG_ERR("HALTING");
+	ui_led_set_pattern(UI_LED_ERROR_UNKNOWN);
+	while (true) {
+		k_cpu_idle();
```

Comment:
Note that this doesn't yield, so that the log message will only be visible if immediate logging is enabled.
`LOG_PANIC()` can be used if it's intentional to not yield, otherwise an added `k_sleep()` will work